assets/src/components/ViewHeader.js

```diff
@@ -0,0 +1,38 @@
+import React, { useEffect, useState } from 'react'
+import Typography from '@material-ui/core/Typography'
+
+function ViewHeader (props) {
+  const [domElement, setDomElement] = useState(null)
+  const [announced, setAnnounced] = useState(false)
+  const { children } = props
+
+  const handleBlur = () => { setAnnounced(true) }
+
+  // Focuses the page header only when the domElement changes.
+  // This should only be when it changes from null to something on initial render.
+  useEffect(() => {
+    if (domElement !== null) {
+      domElement.focus()
+    }
+  }, [domElement])
+
+  const headerProps = {
+    className: 'view-header',
+    variant: 'h5',
```

Comment:
Since the variant mapping to appears as h5->h1 and h6->h2, does it make sense to have this variant as `h5` instead of `h1`?

Reply:
This has been confusing to me, but I think we have to use the heading with the style we want (`h5`) and `variantMapping` will convert the actual tag to the right number.